cdap-docs/examples-manual/source/examples/stream-conversion.rst

```diff
@@ -0,0 +1,163 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform WordCount Application
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _examples-stream-conversion:
+
+==========
+Word Count
+==========
+
+A Cask Data Application Platform (CDAP) Example demonstrating Time-Partitioned File Sets.
+
+Overview
+========
+
+This application receives simple events through a stream, and periodically converts these events into
+partitions of a time-partitioned file set. These partitions can be queried with SQL. These are the
+components of the application:
+
+  - The ``events`` stream receives simple events, where each event body is a number.
+  - The ``converted`` dataset is a time-partitioned file set in Avro format.
+  - The ``StreamConversionMapReduce`` reads the last 5 minutes of events from the stream and writes them to a
+    new partition in the ``converted`` dataset.
+  - The ``StreamConversionWorkflow`` is scheduled every 5 minutes and only runs the
+    ``StreamConversionMapReduce``.
+
+Let's look at some of these components, and then run the Application and see the results.
+
+The Word Count Application
+--------------------------
+
+As in the other :ref:`examples,<examples-index>` the components
+of the Application are tied together by the class ``StreamConversionApp``:
+
+.. literalinclude:: /../../../cdap-examples/StreamConversion/src/main/java/co/cask/cdap/examples/streamconversion/StreamConversionApp.java
+     :language: java
+     :lines: 34-
+
+The interesting part is the creation of the dataset ``converted``:
+
+- It is a ``TimePartitionedFileSet``. This is an experimental new dataset type, introduced in CDAP 2.7.0. This
+  dataset manages the files in a ``FileSet`` by associating each file with a time stamp.
+- The properties are divided in two sections:
+
+  - The first set of properties configures the underlying FileSet, as documented in the
+    :ref:`FileSet,<datasets-fileset>` section.
+  - The second set of properties configures how the dataset is queryable with SQL. Here we can enable the
+    dataset for querying, and if so, we must specify Hive-specific properties for the Avro format: A SerDe,
+    and input and an output format, and an additional table property, namely the schema, for the Avro SerDe.
```

Comment:
a SerDe, input and output formats, and an additional table property, namely the schema for the Avro SerDe.

Is "Avro SerDe" in addition to the first "SerDe". If they are the same, I would reverse the order:

an Avro SerDe, input and output formats, and an additional table property, namely the schema for the SerDe.